actions/cache-node-modules/action.yml

```diff
@@ -7,6 +7,11 @@ inputs:
   build:
     description: 'Extra commands to run if nothing is cached'
     required: false
+    default: 'echo'
+  build_on_cache_fail:
+    description: 'Whether you want to run `npm run --if-present build` if there is no cache
+    require: false
+    default: 'true' # defaulting to true to not break any existing users.
```

Comment:
Does this need to be `'true'` or can it be `true`?

Reply:
Ping @SgtPooki

Reply:
It has to be a string based on current code in this action (it checks for string value of 'true') and [github docs](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault)

Lmk if you know of a way to support booleans for input on github actions


Related:

- https://github.com/actions/runner/issues/1483
- https://github.com/actions/toolkit/issues/361